ports/lpc55/boards.h

```diff
@@ -31,6 +31,9 @@
 
 #include "board.h"
 
+// Flash Start Address of Application
+#define BOARD_FLASH_APP_START  0x10000 
```

Comment:
64KB is quite a lot for this mcu, what is our current output size ? If possible, 32KB is more ideal.

Reply:
With all the debug logs turned on it is already about 32kB.  We can probably get to 32KB, but I would like to keep some headroom for development.

Reply:
the default size should be around 32KB, you can temporarily increases the size with DEUBG LOG build.

Reply:
Maybe trying -Os would help as well, though I think at this state, we could just leave it with 64KB for easy debug. Though I would hope to have it around 32KB for release and working version.